tests/copytests/cases/test035.sh

```
@@ -0,0 +1,13 @@
+# copy an empty directory to a non-existing directory ending with / without -r flag
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)